setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# `.generated-sources/emily/Cargo.toml` changes when the build script runs.
+git update-index --assume-unchanged .generated-sources/emily/Cargo.toml
+# to undo: `git update-index --no-assume-unchanged .generated-sources/emily/Cargo.toml`
```

Comment:
Oh very interesting... I don't have more to add than that.

Reply:
Yeah, it's *blocking* if people don't run this, but they'll want to.

When the openapi generator creates the client it overwrites `.generated-sources/emily/Cargo.toml` with the dependencies of the auto-generated code. Because it's pushed in the repo git won't let you switch to another branch without pushing your changes to it or stashing it, despite it being in the`.gitignore`.

This in itself isn't too bad, but if you're using VS code with the rust analyzer extension it'll automatically run `build.rs` occasionally, meaning you have to race the extension running the build command to switch branches.

The `Cargo.toml` in here is a dummy file, so it's better that it's just ignored, and if we do need to update it the undo command is in the `setup.sh` script.